Index: src/test/java/FirstTest/TextboxTest.java
===================================================================
diff --git a/src/test/java/FirstTest/TextboxTest.java b/src/test/java/LessonTwoHomework/TextboxTest.java
rename from src/test/java/FirstTest/TextboxTest.java
rename to src/test/java/LessonTwoHomework/TextboxTest.java
--- a/src/test/java/FirstTest/TextboxTest.java	
+++ b/src/test/java/LessonTwoHomework/TextboxTest.java	
@@ -1,4 +1,4 @@
-package FirstTest;
+package LessonTwoHomework;
 
 import com.codeborne.selenide.Configuration;
 import org.junit.jupiter.api.BeforeAll;
@@ -15,8 +15,6 @@
     @BeforeAll
     static void setup() {
         Configuration.startMaximized = true;
-
-
     }
 
     @Test
@@ -56,9 +54,5 @@
         $x("//tbody/tr[8]/td[2]").shouldHave(text("logo.png"));
         $x("//tbody/tr[9]/td[2]").shouldHave(text("Kazan"));
         $x("//tbody/tr[10]/td[2]").shouldHave(text("NCR Delhi"));
-
-
     }
-
-
 }
\ No newline at end of file
